ansible/roles/cuda/defaults/main.yaml

```diff
@@ -0,0 +1,2 @@
+autoware_env_dir: /opt/autoware/env
+ros_domain_id: 1
```

Comment:
I'm not sure, but I guess it would be better to make these variables global because they are duplicated.
![image](https://user-images.githubusercontent.com/31987104/208825302-cd982a92-c6a4-4016-b9f3-4ec68f1e831b.png)

Reply:
https://github.com/tier4/perception_ecu_container/pull/1/commits/c3e3b18dd69cfc4531d0687878b1f9b402f3a637